activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Allow schema cache path to be defined in the database configuration file.
```

Comment:
Can you expand this entry with an example?

Reply:
Good idea! Will do.